src/main/java/seedu/address/model/record/Medication.java

```diff
@@ -16,7 +16,7 @@ public class Medication {
      *
      * @param medication Input string.
      */
-    private Medication(String medication) {
+    public Medication(String medication) {
```

Comment:
I think we can use the factory method for any calls to the constructor of Medication.

Reply:
I used the constructor in `EditRecordDescriptorBuilder#WithMedications`, but I agree that using the factory method would be better, will amend.